x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/rollup/job/TermsGroupConfig.java

```diff
@@ -42,27 +43,39 @@
  *     ]
  * }
  */
-public class TermsGroupConfig implements Writeable, ToXContentFragment {
-    private static final String NAME = "term_group_config";
-    public static final ObjectParser<TermsGroupConfig.Builder, Void> PARSER = new ObjectParser<>(NAME, TermsGroupConfig.Builder::new);
+public class TermsGroupConfig implements Writeable, ToXContentObject {
```

Comment:
The other configs should also implement ToXContentObject  and change their `toXContent` but let's do that in a follow up.

Reply:
I agree and I'll take care of this if that's OK. I already have most of the changes in a local branch.